### ISSUE TITLE
Error while using "includeLabelsAndAnnotations" in https://docs.kubecost.com/apis/savings-apis/api-request-right-sizing-v2

### ISSUE THREAD

**Issue by sharanm** [kubecost/cost-analyzer-helm-chart#3337]
### Kubecost Helm Chart Version

2.2.1

### Kubernetes Version

1.27

### Kubernetes Platform

Other (specify in description)

### Description

KUBECOST_ADDRESS='http://localhost:9090/model'

curl -G \
  -d 'algorithmCPU=max' \
  -d 'algorithmRAM=max' \
  -d 'targetCPUUtilization=1' \
  -d 'targetRAMUtilization=1' \
  -d 'window=36h' \
  -d 'includeLabelsAndAnnotations=true' \  --data-urlencode 'filter=namespace:"default"' \
  ${KUBECOST_ADDRESS}/savings/requestSizingV2

**Failed to parse request into request sizing options: invalid HTTP parameters present: [includeLabelsAndAnnotations]**

### Steps to reproduce

1. Install kubecost via Helm
2. Perform port-forward
3. Execute API

### Expected behavior

We should be able to see recommendation along with annoations and labels

### Troubleshooting

- [X] I have read and followed the [issue guidelines](https://github.com/kubecost/cost-analyzer-helm-chart/blob/develop/ISSUE_GUIDELINES.md) and this is a bug impacting only the Helm chart.
- [X] I have searched other issues in this repository and mine is not recorded.

**Comment by jessegoodier** [comment]
Thanks for reporting this. Looks like it is a bug introduced with 2.0. 
Will get it added to the priorities.

**Comment by sharanm** [comment]
Is there a way to workaround this for now? Would version downgrade work? If
yes, which version should I use?

**Comment by jessegoodier** [comment]
yes downgrading to version 1.108.1 without using aggregator (which is not enabled by default) works.

**Comment by sharanm** [comment]
Oh , that would be quite a downgrade, any rough estimate by what time this can eb fixed ?

**Comment by jessegoodier** [comment]
Sorry for that. Agree 2.x is much nicer. :) 

Almost certainly in 2.3, which is about a month.
